### PR TITLE
Updated description for `atan()` function

### LINKED ISSUE #992: [Term Entry] C++ Math Functions: `atan()`
### Reason/inspiration (optional)

Under c++/concepts/math-functions/terms/, create a new term entry for the `atan()` function.

In the terms/ folder, make a new folder called `atan`
Inside the isnan folder, create a new markdown file called `atan.md`
Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
Let's include an "Example" section with a \```cpp code snippet.
Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by codewithdev**
### Description

Closes #992 
The https://www.codecademy.com/resources/docs/cpp/math-functions/atan page describes the use of `atan()` function. This PR adds the following changes to the page:
- Updated function description.
- Added one more example for clarity.

### Type of Change

- Adding a new Example
- Editing the description
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-Returns the inverse tangent of the argument in radians.
+The `<cmath>` header file defines `atan()` function that returns the inverse tangent of an argument in radians. It takes a single argument (positive, negative, or zero) and returns the output in radians.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:14]:
```suggestion
The **`atan()`** function that returns the inverse tangent of an argument (positive, negative, or zero) in radians.
```

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ CatalogContent:
...
 
 ## Syntax
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:17]: On line # 18, let's change \```py to \```pseudo.

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 3]
```diff
@@ -21,19 +21,35 @@ atan(n)
...
   double result;
 
-  result = std::atan(x);
+  result = std::atan(value);
 
   std::cout << result << " radians" << "\n";
   // Output: 1.5708 radians
 }
 ```
+### Example 2
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:40]:
```suggestion
### Codebyte Example
```

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 4]
```diff
@@ -21,19 +21,35 @@ atan(n)
...
+
+The following example uses the `atan()` function that returns the inverse tangent of value `1.0`:
+
+```cpp
+#include <iostream>
+#include<cmath>
+
+int main(){
+  double value= 1.0, result;
+  result= atan(value);
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:50]:
```suggestion
  double value = 1.0, result;
  result = atan(value);
```

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 5]
```diff
@@ -21,19 +21,35 @@ atan(n)
...
+
+  std::cout << result << " radians" << "\n";
+  //Output: 0.785398 radians
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:53]:
```suggestion
  std::cout << result << " radians" << "\n";
```

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 6]
```diff
@@ -21,19 +21,35 @@ atan(n)
...
-  result = std::atan(x);
+  result = std::atan(value);
 
   std::cout << result << " radians" << "\n";
   // Output: 1.5708 radians
 }
 ```
+### Example 2
+
+The following example uses the `atan()` function that returns the inverse tangent of value `1.0`:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:42]:
```suggestion
The following example returns the inverse tangent of the `value` with the `atan()` function:
```

File: content/cpp/concepts/math-functions/terms/atan/atan.md [hunk 7]
```diff
@@ -21,19 +21,35 @@ atan(n)
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/atan/atan.md:21]: Can we write 1-3 sentences here that explains the domain/range of the `n` parameter along with any edge cases (e.g., pos/neg infinity).

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@codewithdev Finished with my review and left you some comments/suggestions. Let me know when you've finished implementing the changes or if you have any questions. 😄

**Comment by Dusch4593** [comment]
@codewithdev Did you have any questions or need any help with implementing the requested changes? Just a reminder that today is the last day to merge your PRs and have them count for Hacktoberfest.

**Comment by Dusch4593** [comment]
@codewithdev Did you have any questions or need any help with implementing the requested changes? Even though Hacktoberfest is over, you can still contribute to Codecademy Docs anytime! 😄

**Comment by SSwiniarski** [comment]
@codewithdev, because it's been a few weeks, I've gone ahead and committed all of @Dusch4593's suggestions in the interest of getting this merged.

**Review by SSwiniarski — APPROVED** [review]
@Dusch4593, this looks good to me now. Anything we need to add?

**Review by Dusch4593 — APPROVED** [review]
@codewithdev @SSwiniarski LGTM for merge! 😄

I made some last edits in f247c1318c2ca6f8c117e9b292b1ccb6a72b3210 that I wanted to note:

- Moved parenthetical statement from line # 14 to line # 22.
- On line # 22:
  - Changed "argument" to "parameter" for the "Syntax" section.
  - Turned last two sentences into list of edge cases.
- Removed " 1" from header on line # 27.
- Allocated output from "Example" to its own \```shell snippet.
- Added `codebyte/cpp` signature to "Codebyte Example" section.
- Moved opening curly brace `{` to same line as `int main()` to be in line with syntax in our C++ courses.

**Comment by Dusch4593** [comment]
Congrats and thank you for contributing to Codecademy Docs! 🥳 🎊 🎉

Your changes should now be live: https://www.codecademy.com/resources/docs/cpp/math-functions/atan

Screen Shot 2022-11-21 at 1 00 36 PM

**Pull request merged**